app/renderer/components/tabs/tabs.js

```diff
@@ -195,6 +196,7 @@ class Tabs extends React.Component {
                 key={'tab-' + frameKey}
                 ref={(node) => this.tabRefs.push(node)}
                 frameKey={frameKey}
+                tabsPerTabPage={this.props.tabsPerTabPage}
```

Comment:
Wondering why a new prop has been added here, but no change to the Tab component to utilize it. Is this fixing a different bug?

Reply:
oh my bad this was for another pr thanks for spotting